### PR TITLE
HTCONDOR-2235 Organize section of user manual into job scheduling

### PR DESCRIPTION
**Pull request by GregThain**
subsection

Also, re-home the "running on different architecture" "Special Environment" to "Submitting a Job"

<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [ ] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [x] Verify that the branch destination of the PR matches the target version of the ticket
- [x] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [x] Check for documentation, if needed  (documentation [build logs](https://readthedocs.org/projects/htcondor/builds/))
- [ ] Check for version history, if needed
- [x] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: docs/users-manual/job-scheduling.rst [hunk 1]
```diff
@@ -1,11 +1,148 @@
...
+
+Job Priority
+''''''''''''
+
+:index:`priority<single: priority; job>` :index:`of a job<single: of a job; priority>`
+
+Job priorities allow a user to sort their own jobs to determine which are
+tried to be run first.  A job priority can be any integer: larger values 
+denote better priority.  So, 0 is a better job priority than -3, and 6 is a better than 5.
+:index:`condor_prio<single: condor_prio; HTCondor commands>`
```
Comment (ColeBollig) [inline docs/users-manual/job-scheduling.rst:27]: This index seems out of place currently. Should we remove it or find a way to better incorporate this via :tool:`condor_prio`?

File: docs/users-manual/job-scheduling.rst [hunk 2]
```diff
@@ -1,11 +1,148 @@
...
+Note that job priorities are computed per user, so that whatever job priorities
+one user sets has no impact at all on any other user, in terms of how many jobs
+users can run or in what order.  Also, unmatchable high priority jobs do not block
```
Comment (ColeBollig) [inline docs/users-manual/job-scheduling.rst:30]: Perhaps a first class `.. note::` here

File: docs/users-manual/job-scheduling.rst (hunk 1 above)
Comment (ColeBollig) [reply docs/users-manual/job-scheduling.rst:27]: Just noticed that we use `:tool:'condor_prio'` later in this section so I think this index should be removed

File: docs/users-manual/job-scheduling.rst [hunk 3]
```diff
@@ -1,11 +1,148 @@
...
+:index:`priority<single: priority; preemption>`
+:index:`priority<single: priority; user>`
+:index:`of a user<single: of a user; priority>`
+
+Slots are allocated to users based upon user priority. A lower
+numerical value for user priority means proportionally better priority, 
+so a user with priority 5 will be allocated 10 times the resources as
+someone with user priority 50. User priorities in HTCondor can be 
+examined with the :tool:`condor_userprio` command.
+:index:`condor_userprio<single: condor_userprio; HTCondor commands>` HTCondor
```
Comment (ColeBollig) [inline docs/users-manual/job-scheduling.rst:67]:
```suggestion
examined with the :tool:`condor_userprio[HTCondor Commands]` command. HTCondor
```

File: docs/users-manual/job-scheduling.rst [hunk 4]
```diff
@@ -1,11 +1,148 @@
...
+priority of each individual user changes according to the number of
+resources the individual is using. Each user starts out with the best
+possible priority: 0.5. If the number of machines a user currently has
+is greater than the user priority, the user priority will worsen by
+numerically increasing over time. If the number of machines is less then
+the priority, the priority will improve by numerically decreasing over
+time. The long-term result is fair-share access across all users. The
+speed at which HTCondor adjusts the priorities is controlled with the
+configuration variable :macro:`PRIORITY_HALFLIFE[definition]` HTCondor
+an exponential half-life value. The
```
Comment (ColeBollig) [inline docs/users-manual/job-scheduling.rst:84]: The tail end of this sentence is wonky

File: docs/users-manual/job-scheduling.rst (hunk 1 above)
Comment (GregThain) [reply docs/users-manual/job-scheduling.rst:25]: Pushed fix.

File: docs/users-manual/job-scheduling.rst (hunk 2 above)
Comment (GregThain) [reply docs/users-manual/job-scheduling.rst]: noted.

File: docs/users-manual/job-scheduling.rst [hunk 5]
```diff
@@ -1,11 +1,148 @@
...
+For instance, if a low priority user is utilizing all available machines
+and suddenly a higher priority user submits jobs, HTCondor may
+vacate jobs belonging to the lower priority user. 
+
+User priorities are keyed on ``<username>@<domain>``, for example
+``johndoe@cs.wisc.edu``. The domain name to use, if any, is configured
+by the HTCondor site administrator. Thus, user priority and therefore
+resource allocation is not impacted by which machine the user submits
+from or even if the user submits jobs from multiple machines.
+:index:`nice job` :index:`nice job<single: nice job; priority>`
```
Comment (ColeBollig) [inline docs/users-manual/job-scheduling.rst:103]:
```suggestion
:index:`nice job<single: nice job; priority>`
```

File: docs/users-manual/job-scheduling.rst (hunk 4 above)
Comment (GregThain) [reply docs/users-manual/job-scheduling.rst:84]: and it is incorrect..  Fixed.

File: docs/users-manual/job-scheduling.rst [hunk 6]
```diff
@@ -1,11 +1,148 @@
...
+
+How Jobs are Vacated
+''''''''''''''''''''
+
+:index:`vacate` :index:`vacate<single: vacate; preemption>`
+
+When HTCondor needs a job to vacate a machine for whatever reason, it
+sends the job an operating system signal specified in the :ad-attr:`KillSig`
+attribute of the job's ClassAd. The value of this attribute can be
+specified by the user at submit time by placing the **kill_sig** option
```
Comment (ColeBollig) [inline docs/users-manual/job-scheduling.rst:123]:
```suggestion
specified by the user at submit time by placing the :subcom:`kill_sig` option
```

File: docs/users-manual/job-scheduling.rst [hunk 7]
```diff
@@ -461,3 +598,131 @@ There are some limitations to HTCondor's job deferral feature.
...
+
+ *condor_status -available*
+    shows only machines which are willing to run jobs now.
+ *condor_status -run*
+    shows only machines which are currently running jobs.
+ *condor_status -long*
+    lists the machine ClassAds for all machines in the pool.
+
+Refer to the :doc:`/man-pages/condor_status` command reference page for a
+complete description of the :tool:`condor_status` command.
```
Comment (ColeBollig) [inline docs/users-manual/job-scheduling.rst:689]: I think we can remove this given all the tool references for condor_status in the section

**Pull request merged**